### DISCUSSION TITLE
官方文档语言切换有问题啊

### DISCUSSION THREAD

**Discussion by elarbor** [hexojs/hexo#5440] [Question / Answer]
![动画](https://github.com/hexojs/hexo/assets/43592730/62b04b8a-ed0b-4a8a-ba51-971eb0d94e6b)

**Comment by stevenjoezhang** [comment]
Access to `/` will always be redirected:
https://github.com/hexojs/site/blob/4adb1d3811af8a59eaedd0c3104a87fe5f101538/netlify.toml#L13-L18

It's a little tricky to resolve because `nf_lang` cookie is not working in such condition:
https://github.com/hexojs/site/commit/f7dee9ae428b27898060face4a836271752cddab

**Comment by uiolee** [comment]
The `about` page is not multilingual
